README.md

```diff
@@ -14,6 +14,7 @@ Contents:
 * [`react`](#react)
 * [`styled-components`](#styled-components)
 * [`whatwg-fetch`](#whatwg-fetch)
+* [`Unspecific Libraries`](#babel-plugin-transform-imports)
```

Comment:
Note: incorrect href

Reply:
I have done this on purpose, as it seems like GitHub can't link to things that contain white-space in it. Or is there a different MD syntax to create an anchor in that case?

Reply:
As far as I see, spaces get replaced with dashes (`-`), like here: https://github.com/lodash/lodash#module-formats

So “Unspecific libraries” would have a link of `#unspecific-libraries`